static/js/filterManager.js

```diff
@@ -1,12 +1,25 @@
 const FilterManager = (function() {
-
+    
+    const FILTERS_BY_DEFAULT = {
+        "country": "United States",
+        "language": "en",
+        "online": "",
+        "format": "",
+        "category": ""
+    }
+    
     function reloadLastFilters() {
         try {
             const filters = ["country", "online", "language", "format", "category"];
-    
             filters.forEach(function(filter) {
                 const lastValue = querySt(filter);
-                if(lastValue) document.getElementById(filter).value = lastValue;
+                if(lastValue != undefined){
```

Comment:
maybe you can replace this with if(lastValue)

Reply:
We use "" when searching for all events. So we compare with undefined in this case